SimPEG/Maps.py

```diff
@@ -669,6 +719,38 @@ def deriv(self, m, v=None):
         return self.P
 
 
+class Projection(IdentityMap):
```

Comment:
we may want to name this differently. All it does is re-arrange / select model parameters